server/routerlicious/packages/lambdas/src/nexus/index.ts

```diff
@@ -1039,66 +1039,61 @@ export function configureWebSocketServices(
 		);
 
 		// Message sent when a new signal is submitted to the router
-		socket.on(
-			"submitSignal",
-			(clientId: string, contentBatches: (IDocumentMessage | IDocumentMessage[])[]) => {
-				// Verify the user has subscription to the room.
-				const room = roomMap.get(clientId);
-				if (!room) {
+		socket.on("submitSignal", (clientId: string, contentBatches: (unknown | unknown[])[]) => {
+			// Verify the user has subscription to the room.
```

Comment:
Isn't `unknown | unknown[]` just `unknown`?
So the only statement here is that contentBatches are an array.
Just use `unknown[]`.
Then add comments.